src/components/ScrollToTop/ScrollToTop.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+
+/**
+ * A component that scrolls to the top of the page on every URL update.
+ */
+export const ScrollToTop: React.FC = (): null => {
+  const history = useHistory();
+
+  // credit goes to https://stackoverflow.com/a/54343182
+  useEffect(() => {
+    const unlisten = history.listen(() => {
+      window.scrollTo({ top: 0, behavior: 'smooth' });
```

Comment:
🙌

Reply:
Yes, my Google-fu skills skyrocketed in the last three months :).